shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -22,41 +22,20 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     with Updatable[ApiFlightWithSplits]
     with WithLastUpdated {
 
-  def totalPaxFromApi: Option[TotalPaxSource] = splits.collectFirst {
+  def bestPaxFromApi: Option[PaxSource] = splits.collectFirst {
```

Comment:
This is now only used in `bestPaxSource` and it's redundant in that function so can be removed